pages/common/katana.md

```diff
@@ -14,7 +14,7 @@
 
 - Use [p]a[s]sive sources (Wayback Machine, Common Crawl, and AlienVault) for URL discovery:
 
-`cat {{example.com}} | katana -passive`
+`cat {{path/to/domains.txt}} | katana -passive`
```

Comment:
```suggestion
`katana -passive < {{path/to/domains.txt}}`
```
Good catch!
A minor suggestion: this is an [Useless Use of Cat (UUOC)](https://porkmail.org/era/unix/award#cat), which leaves to some slowdowns like an unnecessary process fork, the file being read by cat and by katana too. This gets worse the larger the file is.